cves/CVE-2011-2729.yml

```diff
@@ -43,7 +43,7 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Jsvc is an abstraction layer that allows Java applications to run UNIX more easily.  The vulnerablity in this service can allow applications to access files belonging to superuser (this could be password files, superuser logs, sysystem configuration, etc.)
```

Comment:
Did you mean "run **on** UNIX more easily."?

Reply:
Thanks for finding that grammar mistake I'll make sure to correct it.